shared.js

```diff
@@ -7,6 +7,16 @@ module.exports = {
     PATHS,
   },
 
+  VKUI_TOKENS_CSS: [
+    './node_modules/@vkontakte/vkui-tokens/themes/vkBase/cssVars/declarations/onlyVariables.css',
+    './node_modules/@vkontakte/vkui-tokens/themes/vkBase/cssVars/declarations/onlyVariablesLocal.css',
+    './node_modules/@vkontakte/vkui-tokens/themes/vkBaseDark/cssVars/declarations/onlyVariablesLocal.css',
+    './node_modules/@vkontakte/vkui-tokens/themes/vkIOS/cssVars/declarations/onlyVariablesLocal.css',
+    './node_modules/@vkontakte/vkui-tokens/themes/vkIOSDark/cssVars/declarations/onlyVariablesLocal.css',
+    './node_modules/@vkontakte/vkui-tokens/themes/vkCom/cssVars/declarations/onlyVariablesLocal.css',
+    './node_modules/@vkontakte/vkui-tokens/themes/vkComDark/cssVars/declarations/onlyVariablesLocal.css',
+  ],
```

Comment:
```suggestion
  ].map(path => require.resolve(`../../${path}`)),
```
может сразу абсолютные пути хранить вот так?

Reply:
Нам в `/postcss.config.js` нужны именно такие пути